eventhubs-streamanalytics-eventhubs/tools/README.md

```diff
@@ -1,11 +1,41 @@
-# Testing End-To-End Latency
+---
+page_type: sample
+languages:
+- csharp
+products:
+- azure
+- azure eventhubs 
+extensions:
+- platforms: dotnet
+---
 
-Get the Event Hub connection string created and displayed during the script execution. If you cannot find it, just grab the execution string for the `Listen` Shared Access Policy for the output eventhub.
+# Getting started on processing received message from eventhub #
 
+Azure eventhubs-streamanalytics-eventhubs sample for processing received message from eventhub -
+ - Create an EventHubConsumerClient based on connection string for the `Listen` or `Manage` Shared Access Policy for the eventhub.
+ - Read events from EventHubConsumerClient.
+ - Processing received message and write to a .csv file.
+
+# Running this Sample #
+To run this sample:
+
+If you don't have an Azure subscription, create a [free account] before you begin.
+	
+Go to [Azure portal], sign in with your account. You can find how to [create event hub service] from here. 
+
+Get the EventHubConnectionString for the `Listen` or `Manage` Shared Access Policy for the eventhub.
+
+```bash	
+git clone https://github.com/Azure-Samples/streaming-at-scale.git
+```
 From within the `eh-asa-perform` folder run:
 
 ```bash
-dotnet run -- -c "<connection string>"
+dotnet run -- -c "<EventHubConnectionString>"
 ```
+The application will read from the eventhub and will measure latency (difference from Enqueued and Processed time) for each batch of recevied messages.
```

Comment:
If you're referring to a single event hub within the namespace, then you'll want a space here.   If you're referring to the product, you'll want "Event Hubs."   It's a bit unclear to me, but I think the former is what you're after.

Reply:
space added, "event hub"